common/src/main/java/tachyon/Constants.java

```diff
@@ -66,9 +66,14 @@
   public static final int DEFAULT_HOST_RESOLUTION_TIMEOUT_MS = 5000;
 
   public static final String BLOCK_MASTER_SERVICE_NAME = "BlockMaster";
+  public static final long BLOCK_MASTER_SERVICE_VERSION = 1;
```

Comment:
Maybe there should be a comment above this block briefly explaining that the versions should be incremented on incompatible RPC changes?